__tests__/data/vero_router_input.json

```
@@ -0,0 +1,88 @@
+[
+  {
+    "destination": {
+      "Config": {
+        "authToken": "KDH4JNDHCFJHJ57SJOWJE490W01JFNHGDSSFHDKDSDF"
+      }
+    },
+    "metadata": {
+      "jobId": 1
+    },
+    "message": {
+      "context": {
+        "traits": {
+          "email": "user001@tech.com"
+        }
+      },
+      "type": "Identify",
+      "userId": "user001"
+    }
+  },
+  {
+    "destination": {
+      "Config": {
+        "authToken": "KDH4JNDHCFJHJ57SJOWJE490W01JFNHGDSSFHDKDSDF"
+      }
+    },
+    "metadata": {
+      "jobId": 1
+    },
+    "message": {
+      "context": {
+        "traits": {
+          "email": "user002@tech.com"
+        }
+      },
+      "type": "Identify",
+      "anonymousId": "b4ffheww8eisndbdjgdewifewfgerwibderv"
+    }
+  },
+  {
+    "destination": {
+      "Config": {
+        "authToken": "KDH4JNDHCFJHJ57SJOWJE490W01JFNHGDSSFHDKDSDF"
+      }
+    },
+    "metadata": {
+      "jobId": 1
+    },
+    "message": {
+      "context": {
+        "traits": {
+          "address": "Caravela Beach Goa",
+          "homwTown": "Mawsynram",
+          "email": "user005@tech.com"
+        }
+      },
+      "integerations": {
```

We should fix this typo @sandeepdsvs 
```suggestion
      "integrations": {
```

---

Done